### PR TITLE
Add instructions to get your near account id and private key

### PR DESCRIPTION
**Pull request by Lukisong82**
Very crucial instructions for people who are very new to web3 development world.
Also very new to the Near ecosystem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,3 +91,14 @@ Before using NEAR-CA, ensure you have the following environment variables set:
 - `NEAR_MULTICHAIN_CONTRACT`: The NEAR contract that handles multichain operations.
 
 Copy the `.env.example` file and place these values in `.env`
+
+
+Steps to get your NEAR_ACCOUNT_ID and NEAR_ACCOUNT_PRIVATE_KEY:
```
Comment (bh2smith) [inline README.md:96]:
```suggestion
Steps to get your `NEAR_ACCOUNT_ID` and `NEAR_ACCOUNT_PRIVATE_KEY`:
```

File: README.md [hunk 2]
```diff
@@ -91,3 +91,14 @@ Before using NEAR-CA, ensure you have the following environment variables set:
...
+
+1. Create a mintbase wallet, super easy, here: https://wallet.mintbase.xyz/
+2. Your `XYZ.near` is your `NEAR_ACCOUNT_ID`.
```
Comment (bh2smith) [inline README.md:99]:
```suggestion
2. Your `XYZ.testnet` is your `NEAR_ACCOUNT_ID`.
```

**Review by bh2smith — COMMENTED** [review]
Hey thanks for suggesting these instructions! I made a couple suggestions (one just adding backticks) and the other changing to testnet (since this is currently a testnet only software package).

I am checking with the team about whether these instructions exist somewhere already (so we could simply reference them from here) because I don't think that specific guides on how to recover a private key from a wallet really belong here. I do very much like this wording and would like to have this land in the appropriate place!

Please hang tight while I find the right place to put this!

**Comment by bh2smith** [comment]
Hey @Lukisong82 - do you plan on coming back to this or should I move it into a TODO issue elsewhere. 

I'll close it at the end of the week if I don't hear from you.

**Comment by Lukisong82** [comment]
> Hey thanks for proposing these instructions! I've made a couple suggestions (one just adding `backticks`) and the other changing to `testnet` (since this is currently a **testnet only** software package).
> 
> I think that these instructions would be much more comfortable living on our docs page: https://docs.mintbase.xyz/
> 
> Notice that there is a section for "wallet" and here is the corresponding
> 
> GitHub Repo: https://github.com/Mintbase/gitbook-docs/tree/main/wallet
> 
> would you mind creating a new page in the docs repo for exporting your key? Then this PR can become a reference to the docs.

Hi @bh2smith, I have made the proposed changes on the readme file. Unfortunately I do not have access to alter the docs.

**Comment by bh2smith** [comment]
Alright, in favour of having the instructions here for now (although they don't really belong here) and a bit of duplication with the content in https://github.com/Mintbase/near-ca/blob/main/examples/README.md. 

Thanks for your contribution 🎈  and have a nice weekend!

**Review by bh2smith — APPROVED** [review]

**Pull request merged**